test/600_proxy_docker_py_test.sh

```diff
@@ -13,7 +15,7 @@ if docker_on $HOST1 run \
     -e DOCKER_HOST=tcp://172.17.42.1:12375 \
     -v /tmp:/tmp \
     -v /var/run/docker.sock:/var/run/docker.sock \
-    joffrey/docker-py python tests/integration_test.py ; then
+    joffrey/docker-py py.test tests/integration_test.py ; then
```

Comment:
I think this should have been updated in line with https://github.com/weaveworks/weave/commit/54833195edf044b8f1f97c7d9de6757c52edccda